src/attr/_make.py

```diff
@@ -184,28 +183,33 @@ class MyClassAttributes(tuple):
 
 def _transform_attrs(cls, these):
     """
-    Transforms all `_CountingAttr`s on a class into `Attribute`s and saves the
-    list in `__attrs_attrs__`.
+    Transform all `_CountingAttr`s on a class into `Attribute`s and save the
+    list in `__attrs_attrs__` while potentially deleting them from *cls*.
```

Comment:
Do we need the 'potentially' there?

Reply:
I think yes because in some cases they’re not there. :)